### PR TITLE
Only import new RPM signing key when installing A7

### PR DESCRIPTION
**Pull request by albertvaka**
We are only going to sign Agent 7 with the new key.

Testing done
------

Manually tested on CentOS and SUSE.

### CODE REVIEW

**Review by KSerrania — DISMISSED** [review]
LGTM, also tested A6 and A7 installs on a CentOS and SUSE box and it worked.

**Comment by albertvaka** [comment]
Let's merge this after the freeze (since it goes live right after merging).

**Comment by albertvaka** [comment]
I had to rebase this because it conflicted with your SUSE11 changes @KSerrania. Can you check my merge makes sense?

File: cmd/agent/install_script.sh
Comment (KSerrania) [inline cmd/agent/install_script.sh]: This could also be wrapped by a conditional to check if we're installing Agent 7:
```bash
  if [ $dd_agent_major_version -ne 7 ]; then
    $sudo_cmd curl -o /tmp/DATADOG_RPM_KEY.public https://yum.${repo_url}/DATADOG_RPM_KEY.public
    $sudo_cmd rpm --import /tmp/DATADOG_RPM_KEY.public
    gpgkeys="https://yum.${repo_url}/DATADOG_RPM_KEY.public\n       https://yum.${repo_url}/DATADOG_RPM_KEY_E09422B3.public"
  else
    gpgkeys="https://yum.${repo_url}/DATADOG_RPM_KEY_E09422B3.public"
  fi
```

Comment (albertvaka) [reply cmd/agent/install_script.sh]: I was trying to avoid so many nested ifs, I think I managed to simplify it a bit in my last commit.

**Review by KSerrania — APPROVED** [review]
LGTM, I started a new pipeline with kitchen tests to make sure this works, let's merge once the tests are green.

**Pull request merged**
